<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vinitas-Air.sthlm.int" ignored="0" name="Test" tests="5" failures="3" timestamp="2022-05-03T21:57:41 CEST" time="10.714" errors="0">
  <testcase name="CreateUser" time="5.283" classname="com.API.tests.RestAPITest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [1141004762] but found [74941],
	expected [AA00BBBB111111] but found [AA00BBBB1111111],
	expected [SEK] but found [Swedish Kronor],
	expected [34303] but found [343.03]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [1141004762] but found [74941],
expected [AA00BBBB111111] but found [AA00BBBB1111111],
expected [SEK] but found [Swedish Kronor],
expected [34303] but found [343.03]
at com.API.tests.RestAPITest.CreateUser(RestAPITest.java:132)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- CreateUser -->
  <testcase name="RetrivePayoutDetailswithBlankId" time="1.127" classname="com.API.tests.RestAPITest">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;500&amp;gt; but was &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:499)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:684)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:127)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at com.API.tests.RestAPITest.RetrivePayoutDetailswithBlankId(RestAPITest.java:154)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- RetrivePayoutDetailswithBlankId -->
  <testcase name="VerifyPayoutsAPIwithInvalidResource" time="1.011" classname="com.API.tests.RestAPITest"/>
  <testcase name="VerifyResponseProperty_PayoutAPI" time="1.145" classname="com.API.tests.RestAPITest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [1141004762] but found [74941],
	expected [AA00BBBB111111] but found [AA00BBBB1111111],
	expected [SEK] but found [Swedish Kronor],
	expected [34303] but found [343.03],
	expected [2022-05-22] but found [May 22nd 2022]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [1141004762] but found [74941],
expected [AA00BBBB111111] but found [AA00BBBB1111111],
expected [SEK] but found [Swedish Kronor],
expected [34303] but found [343.03],
expected [2022-05-22] but found [May 22nd 2022]
at com.API.tests.RestAPITest.VerifyResponseProperty_PayoutAPI(RestAPITest.java:61)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- VerifyResponseProperty_PayoutAPI -->
  <testcase name="VerifyStatuscodeMessageforPayoutAPI" time="1.159" classname="com.API.tests.RestAPITest"/>
</testsuite> <!-- Test -->
